### PR TITLE
Alert logic and update instance types

### PR DESCRIPTION
**Pull request by jpdhas**

### CODE REVIEW

File: ansible/userdata/cloud-config-etcd [hunk 1]
```diff
@@ -40,6 +40,36 @@ coreos:
...
+        ExecStart=/usr/bin/docker run \
+          --name=al-agent-container \
+          --label "app=al-agent-container" \
+          --net=host \
+          --privileged \
+          --mount type=bind,source=/var/run/docker.sock,target=/var/run/docker.sock \
+          --mount type=bind,source=/proc,target=/host/proc \
+          --memory-reservation=100m \
+          --memory=500m \
+          --cpus=1 \
```
Comment (tamas-molnar) [inline ansible/userdata/cloud-config-etcd]: It really needs 1 CPU?

File: ansible/userdata/cloud-config-etcd [hunk 2]
```diff
@@ -40,6 +40,36 @@ coreos:
...
+          --restart=always \
+          alertlogic/al-agent-container:latest
```
Comment (tamas-molnar) [inline ansible/userdata/cloud-config-etcd:68]: Still no versioning?

Comment (jpdhas) [reply ansible/userdata/cloud-config-etcd:68]: https://github.com/Financial-Times/content-alertlogic-helm-config/blob/master/helm/alert-logic/values.yaml#L8 - we are using the latest version of alert logic for our daemonsets. Thought we should be consistent

Comment (tamas-molnar) [reply ansible/userdata/cloud-config-etcd:68]: I know, but the questions still remains - for all other services  we use versioned images so we are in control of what is running in our cluster.

File: ansible/userdata/cloud-config-etcd (hunk 1 above)
Comment (jpdhas) [reply ansible/userdata/cloud-config-etcd]: This is what was recommended by Alert logic themselves. To stick with a minimum CPU of 1 and go till a maximum of 3

Comment (JermilaPaulDhas) [reply ansible/userdata/cloud-config-etcd]: Though i guess since the etcd instance has just 2 cpus, we can try restricting it further and monitor it

Comment (tamas-molnar) [reply ansible/userdata/cloud-config-etcd]: What machines do they think we are running these on? `t3.medium` has 2 vCPU, are we supposed to dedicate half of computing power to AlertLogic?

File: ansible/userdata/cloud-config-etcd (hunk 2 above)
Comment (jpdhas) [reply ansible/userdata/cloud-config-etcd:68]: Could not really see any Dockerfile. Does not really tell me much about what the image has: https://github.com/alertlogic/al-agent-container/tree/master/docker & https://hub.docker.com/r/alertlogic/al-agent-container/.

It is a intrusion detection service. I guess they want to keep things secret -_-

File: ansible/userdata/cloud-config-etcd (hunk 1 above)
Comment (jpdhas) [reply ansible/userdata/cloud-config-etcd]: the general recommendation was 1. But i guess we can always reduce it 🤷‍♀️

Comment (jpdhas) [reply ansible/userdata/cloud-config-etcd]: done @tamas-molnar

**Review by tamas-molnar — APPROVED** [review]

**Review by tamas-molnar — APPROVED** [review]

**Pull request merged**